sherpa/astro/instrument.py

```diff
@@ -1234,11 +1234,16 @@ def get_kernel(self, data, subkernel=True):
         return dataset
 
 
-def create_arf(elo, ehi, specresp=None, exposure=None, ethresh=None, name='user-arf'):
+def create_arf(elo, ehi, specresp=None, exposure=None, ethresh=None,
+               name='user-arf', header=None):
     """Create an ARF.
 
     .. versionadded:: 4.10.1
 
+    .. versionchanged:: 4.13.2
```

Comment:
See version changed conversation a line 1302. The information is better placed in change logs/release notes. One reason is to avoid mismatches or incorrect information (for example when code changes are delayed or subsequently modified and notes aren't updated. In this case, we are not expecting to have a 4.13.2 release. The release schedule was modified to align with the CIAO beta2 schedule so the upcoming release is 4.14.0.